packages/fluentui/docs/src/prototypes/SearchPage/SearchPage.tsx

```diff
@@ -81,7 +81,6 @@ class SearchPage extends React.Component<SearchPageState, any> {
 
         <p>
           <Input
-            ref="input"
```

Comment:
Strings can be used only for deprecated refs API: https://reactjs.org/docs/refs-and-the-dom.html#legacy-api-string-refs. But there is no such usage 🧛